athenz/resource_group.go

```diff
@@ -190,6 +215,21 @@ func resourceGroupUpdate(ctx context.Context, d *schema.ResourceData, meta inter
 		membersToDelete = append(membersToDelete, expandGroupMembers(oldVal.Difference(newVal).List())...)
 		membersToAdd = append(membersToAdd, expandGroupMembers(newVal.Difference(oldVal).List())...)
 	}
+
+	if d.HasChange("tags") {
+		isGroupChange = true
+		_, n := d.GetChange("tags")
+		tags := expandTagsMap(n.(map[string]interface{}))
+		group.Tags = tags
+	}
+
+	if isGroupChange {
```

Comment:
why do we set this flag only when tag changed and not in members? I am confused
or updateGroupMembers take care of that and then the parameter name should be isGroupTagsChanged or something like that